File: app/src/at-tamle/java/asiantech/internship/winter/ex8/reversible.java
package asiantech.internship.winter.ex8;
 //liet ke all so thuan nghich 6 chu so
public class reversible {
    public static boolean process(int n) {
        StringBuilder k = new StringBuilder();
        String str = "" + n;
        k.append(str);
        String check = "" + k.reverse();
        if (str.equals(check))
            return true;
        else
            return false;
    }

    public static void main(String[] args) {
        int n, count = 0;
        for (n = 100000; n <= 999999; n++) {
            if (process(n)) {
                System.out.println(n);
                count++;
            }
        }
        System.out.println("Co " + count + " so thuan nghich co 6 chu so");
    }

}

File: app/src/at-tamle/java/asiantech/internship/summer/kotlin/Reversible.kt
package asiantech.internship.summer.kotlin

object Reversible {
    fun process(n: Int): Boolean {
        val k = StringBuilder()
        val str = "" + n
        k.append(str)
        val check = "" + k.reverse()
        if (str == check)
            return true
        else
            return false
    }

    @JvmStatic
    fun main(args: Array<String>) {
        var n: Int
        var count = 0
        n = 100000
        while (n <= 999999) {
            if (process(n)) {
                println(n)
                count++
            }
            n++
        }
        println("Co $count so thuan nghich co 6 chu so")
    }

}
